backend/routes/userRouter.js

```diff
@@ -0,0 +1,41 @@
+// Base requires:
+const express = require('express');
+const router = express.Router();
+
+// App requires/middleware
+const user = require('../data/helpers/userModel');
+
+
+/* ---------- Endpoints for /api/user ---------- */
+
+/* GET (list) */
+router.get( '/', (req, res) => {
```

Comment:
Shouldn't this have `id` in the url
```suggestion
router.get( '/:id', (req, res) => {
```